docs/dev/json.rst

```diff
@@ -175,3 +175,62 @@ submitted via the ``additional_info`` element. We're open to see how people
 will make use of these unspecified fields and may decide to formally specify or
 scrub elements in the future.
 
+Artist Statistics JSON
+----------------------
+The JSON documents returned from our API look like the following::
+
+   {
+     "payload": {
+        "user_id": "-- the MusicBrainz ID of the user --",
+        "artist": {
+           "all_time": {
+              "artists": [
+                 "-- Artist stats here --"
+              ]
+           },
+           "count": "-- Number of unique artists the user has listened to --"
+         },
+         "last_updated": "-- Date when the statistics were last updated --"
+       }
+     }
+   }
+
+An sample response from the endpoint may look like::
+
+   {
+     "payload": {
+       "artist": {
+         "all_time": {
+           "artists": [
+               {
+                  "artist_mbids": [93e6118e-7fa8-49f6-9e02-699a1ebce105],
+                  "artist_msid": "d340853d-7408-4a0d-89c2-6ff13e568815",
+                  "artist_name": "The Local train",
+                  "listen_count": 385
+               },
+               {
+                  "artist_mbids": [ae9ed5e2-4caf-4b3d-9cb3-2ad626b91714],
+                  "artist_msid": "ba64b195-01dd-4613-9534-bb87dc44cffb",
+                  "artist_name": "Lenka",
+                  "listen_count": 333
+               },
+               {
+                  "artist_mbids": [cc197bad-dc9c-440d-a5b5-d52ba2e14234],
+                  "artist_msid": "6599e41e-390c-4855-a2ac-68ee798538b4",
+                  "artist_name": "Coldplay",
+                  "listen_count": 321
+               },
+            ]
+         },
+         "count": 3
+       },
+       "last_updated": "Sun, 03 May 2020 08:26:01 GMT",
+       "user_id": "John Doe"
+     }
+   }
+
+.. note::
+   `artist_mbids` and `artist_msid` are optional fields and may not be present in all the responses   
+
+.. warning::
+   The statistics API endpoint is still in beta and the `JSON` format may change in the future.
```

Comment:
I'm happy to keep the beta tag, but let's try not to change the the json format, otherwise clients have no real incentive to actually use the endpoint.

Let's remove the format change warning